File: Metro-Project/src/main/java/com/springboot/model/Booking.java
package com.springboot.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name="booking")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
public class Booking {
	@Id @GeneratedValue @Column(name = "booking_id")
 private int bookingId;
	@Column(length = 50)
 private String email;
	@Column(name = "no_of_ticket")
 private int noofTicket;
	@Column(length = 20 ,name="fromstation")
 private String from;
	@Column(length = 20,name="tostation")
 private String to;
	@Column(name = "totalfare")
 private double totalFare;
 
}

File: src/app/bookingsummary-com/Booking.ts
export class Booking {
    bookingId: number;
    email: String;
    noofTicket: number;
    from: String;
    to: String;
    totalFare: number;

    constructor(bi: number, e: string, nt: number, f: String, t: String, tf: number) {
        this.bookingId = bi;
        this.email = e;
        this.noofTicket = nt;
        this.from = f;
        this.to = t;
        this.totalFare = tf;
    }


}